Updated roles and perms to access laps (#21241)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/devicelocalcredentialinfo-get.md

|Permission type|Permissions (from least to most privileged)|
|:---|:---|

> |Delegated (work or school account)|DeviceLocalCredential.Read.All,DeviceLocalCredential.ReadBasic.All|

Reviewer: Least to most privileged order.
```suggestion
|Delegated (work or school account)|DeviceLocalCredential.ReadBasic.All, DeviceLocalCredential.Read.All|
```

|Delegated (personal Microsoft account)|Not supported.|

> |Application|DeviceLocalCredential.Read.All,DeviceLocalCredential.ReadBasic.All|
> 
> Note that in order to access the actual passwords on the device, done by including `$select=credentials` as part of the query parameters, the permission DeviceLocalCredential.Read.All is required and DeviceLocalCredential.ReadBasic.All will be insufficient.

Reviewer: Least to most privileged. I also recommend
```suggestion
|Application|DeviceLocalCredential.ReadBasic.All, DeviceLocalCredential.Read.All|

To access the actual passwords on the device, done by including `$select=credentials` as part of the query parameters, the app must be assigned the *DeviceLocalCredential.Read.All* permission and *DeviceLocalCredential.ReadBasic.All* is insufficient.
```
